src/components/BalanceList.js

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components';
+import Balance from './Balance.js';
+
+export default function BalanceList({
+  expenses,
+  setExpenses,
+  setEditExpense,
+  members,
+  expenseFormatter,
+}) {
+  const totalExpenses = Number(
+    expenses
+      .map(expense => expense.amount)
+      .reduce((prev, curr) => +prev + +curr, 0)
+    //The unary plus operator (+prev) converts the strings to numbers
+  );
+
+  return (
+    <Wrapper>
+      <TotalExpenses>
+        <p>Total expenses:</p>
+        <p>{expenseFormatter(totalExpenses)}</p>
+      </TotalExpenses>
+      <Scrollarea>
+        {members.map((member, index) => (
+          <Balance
+            key={index}
```

Comment:
Maybe there is another value for the keys available which is unique and better than the index?